src/sidebar/ExampleSidebar.tsx

```diff
@@ -5,33 +5,38 @@
 
 // https://github.com/excalidraw/excalidraw/blob/4dc4590f247a0a0d9c3f5d39fe09c00c5cef87bf/examples/excalidraw
 
-import { useState } from "react";
```

Comment:
Please make sure to rebase to latest main branch, this file should no longer exist ;)